script/Arcueid/Day 9/09_00_ARC09_4_1.txt

```diff
@@ -69,7 +69,7 @@ Nii-san, what do you want to do about dinner?"
 -- 「……そっか。いや、琥珀さんには悪いけど、食べられそうにない。今夜はこのまま眠らせてくれ」
 "Oh, right...
  Well, give my apologies to Kohaku-san, but I don't think I can stomach eating anything right now.
- I'll be better off going to sleep as-is tonight."
+ I'll be better off going to sleep as is tonight."
```

Comment:
Is this not an adj phrase -> hyphenated?

Reply:
I just based it off MW
![image](https://github.com/Tsukihimates/Tsukihime-Translation/assets/90473809/d4f7e6c7-2df0-4bc5-8714-3ffb7cc08eae)

Reply:
I dunno man it's English there are no rules

Reply:
lol touche lets just go hyphenated